### ISSUE TITLE
atom.workspaceView is deprecated.

### ISSUE THREAD

**Issue by victorhooi** [mpeterson2/save-session#49]
When I start Atom (0.174.0), I get the following warnings:

atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.
```
Atom.Object.defineProperty.get (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/src/atom.js:55:11)
Object.addListeners (/Users/victorhooi/.atom/packages/save-session/lib/project.coffee:34:9)
```

**Comment by mpeterson2** [comment]
I'm not really sure what the new version of this would be. Neither `atom.views.getView(atom.workspace)` nor `atom.workspace` seem to have an `Emitter`.

I've also tried:
``` 
atom.workspace.emitter.preempt 'application:new-window', =>
  console.log 'new window...'
```

but the event didn't seem to get triggered.

Anyone have ideas?

**Comment by mpeterson2** [comment]
Thanks to SalGnt, this issue should be resolved in version `0.11.5`. Let me know if it keeps occurring.
